css/style.css

```diff
@@ -1,19 +1,219 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ body{
+  width: 100%;
+  margin: 0px;
+  display: block;
+ }
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+* {
+  box-sizing: border-box;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+html {
+  font-family: sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 5rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  background-color: rgba(196, 196, 196, 0.1);
+}
+
+.logo1 {
+    width: 50%;
+}
+
+.img{
+    padding-top: 1rem;
+}
+
+.list {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  align-content: centre;
+  list-style: none;
+  padding: auto;
+  gap: 1rem;
+}
+
+.list a{
+  text-decoration: none;
+}
+
+.list :nth-child(1) {
```

Comment:
I really like this use of a pseudoclass selector, it's a very good use case for it.